src/components/input-field/input-field.scss

```diff
@@ -85,9 +85,13 @@
     width: calc(100% - #{functions.pxToRem(20)});
     z-index: z-index.$input-field--formatted-value;
 
-    display: none;
+    position: absolute;
+    top: 0.9rem;
+    pointer-events: none;
+
+    opacity: 0;
```

Comment:
Note to self (for after hackday): This should probably have a no-speak attribute too, because a screenreader should only need to see the actual input and the value present there.